server/app.ts

```diff
@@ -31,6 +34,7 @@ export default function createApp(services: Services): express.Application {
   app.use(setUpWebRequestParsing())
   app.use(setUpStaticResources())
   nunjucksSetup(app, services.applicationInfo)
+  app.use(calendarPreview(services)) // TODO - remove
```

Comment:
Adding calendar preview page route here temporarily so it can be viewed unauthenticated